IMAGES.rst

```diff
@@ -416,3 +416,22 @@ signals). This entrypoint works as follows:
 
 * If first argument is equal to "bash" - you are dropped in bash shell.
 * If there are any arguments they are passed to "airflow" command
+
+Docker Compose
```

Comment:
Should we add this documentation to docs/*.rst?  Otherwise this documentation will not be available to end users.